src/Symfony/Component/DependencyInjection/Compiler/AbstractRecursivePass.php

```diff
@@ -194,7 +194,7 @@ protected function getReflectionMethod(Definition $definition, string $method)
 
     private function getExpressionLanguage(): ExpressionLanguage
     {
-        if (null === $this->expressionLanguage) {
+        if (!isset($this->expressionLanguage)) {
```

Comment:
why not nullable?

Reply:
the private scope confused me. But from #41924 i figured this lazy loading tactic is probalby slightly preferred, as we dont need to give in on type info.

:+1:

Reply:
Right. `null` was previously used for lazy initialization only and I'd like to avoid making properties nullable. If someone accesses this property before initialization, they made a mistake and PHP will raise an exception for us. This is a good thing.